src/frontend/sharedComponents/FormattedData.js

```diff
@@ -0,0 +1,154 @@
+// @flow
```

Comment:
This seems useful! Can we abstract this out as part of this PR, perhaps to @mapeo/react-components? It would be great to re-use this in Md (and get the formatted preset names, too) for the Report. Come to think of it, the Report and the Share screen ought to probably share a lot of code... :)

Reply:
I started this here. https://github.com/digidem/mapeo-components/pull/1

In the desktop app I copy/pasted the relevant two components (FormattedFieldProp & FormattedValue) to get the Report PDF branch ready to launch. We probably should just do an extra piece of work (1-2 weeks) sometime this fall/winter to refactor these components into a shared library.

Reply:
Yes, wrote this in this way as a first step towards extracting into a shared library, see https://github.com/digidem/mapeo-mobile/issues/460 however this depends on #459 which I would rather not do as part of this PR.